CHANGELOG_UNRELEASED.md

```diff
@@ -21,6 +21,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - in `fingraph.v`, new lemmas: `fcard_gt0P`, `fcard_gt1P`
 - in `finset.v`, new lemmas: `properC`, `properCr`, `properCl`
 - in `ssrnat.v`, new lemmas: `subn_minl`, `subn_maxl`
+- in `ssrnat.v`, new lemma: `oddS`
```

Comment:
Isn't it better to merge these two items?

Reply:
I prefer this is done in the release phase, to minimize possible faulty merge union